R/pkg/R/utils.R

```diff
@@ -334,18 +334,21 @@ getStorageLevel <- function(newLevel = c("DISK_ONLY",
                                          "MEMORY_ONLY_SER_2",
                                          "OFF_HEAP")) {
   match.arg(newLevel)
+  strageLevelClass <- "org.apache.spark.storage.StorageLevel"
```

Comment:
Yeah but the variable name spelling is strange right now as `strage` -- could you make the variable name `storage` instead ?